MetabolicAtlas/MetabolicAtlas
# feat: multiple overlay

**Related issue(s) and PR(s)**  
This PR closes private issue 138.

Joint work with @inghylt and @e0.



**Type of change**  

 
**List of changes made**  

Adds possibility to show overlays of different components


**Screenshot of the fix**  

![2022-08-30-144945_1342x849_scrot](https://user-images.githubusercontent.com/1029190/187440884-57c3afcd-787a-45e1-9c90-ab0eea42768c.png)


**Testing**  

- visit eg `explore/Human-GEM/map-viewer/acyl_coa_hydrolysis?dim=2d&panel=1&sel=&search=&coords=-3487.78,-9141.27,0.13,315.43,47.67,-69.89&dataTypes=transcriptomics,fluxomics&dataSources=hpaRna.tsv,HPA_single-cell_reactions.tsv&dataSets=rectum,brain_excitatory%20neurons_0` and verify that you can see the colored overlay for both transcriptomics and fluxomics on the map. In the `Data Overlay` panel, on the right, you should see two boxes, one for each data type.
- verify that you can change data set (Levels) for any of the data types
- verify that you can remove any of the overlay by clicking the cross of the `Overlay` box
- verify that you can reload the page
- try uploading custom data. For for fluxomics (map *Alkaloids biosynthesis*), you can try

 ```
id  tissue 1    tissue 2
MAR01591    0.6 0.3
MAR00447    0.55    0.9
```

- verify that the overlay works in both 3D and 2D.


**Definition of Done checklist**
